csrc/python_frontend/python_bindings.cpp

```diff
@@ -677,48 +677,65 @@ void initNvFuserPythonBindings(PyObject* module) {
   heuristic_params.def(
       "__repr__", [](const HeuristicParams& self) { return self.toString(); });
 
+#define INITPARAMS(cfg, internal_type)                                     \
+  py::class_<internal_type, HeuristicParams> cfg(nvfuser, #internal_type); \
+  cfg.def(py::init());                                                     \
+  cfg.def(                                                                 \
+      "__repr__", [](const internal_type& self) { return self.toString(); });
+#define PARAM(cfg, internal_type, type, name)        \
```

Comment:
TODO: we should probably add a `.doc("foo")` to these params and to the "_config" type to make them more user-friendly